tests/vcr_cassettes/test_list.yaml

```diff
@@ -31,7 +31,7 @@ interactions:
       x-ms-version:
       - '2019-07-07'
     method: PUT
-    uri: https://terrytest2.blob.core.windows.net/pypi/mypkg/pkg.tar.gz
+    uri: https://terrytest2.blob.core.windows.net/pypi/mypkg/pkg.tar.gz?comp=blocklist
```

Comment:
@stevearc how did you generate these three `vcr_cassettes`? if i set the mode to [`all`](https://vcrpy.readthedocs.io/en/latest/usage.html#all), no updates to the files occur I guess because we're pointing to a non existent container URL:
https://github.com/stevearc/pypicloud/runs/7021821492?check_suite_focus=true#step:4:1896

if you have access to azure (I don't) and if that is needed to regenerate for the multipart requests, could you support here?

I've temporarily added my (unreleased) feature branch of `smart_open` to the requirements here so I could properly update the tests, but have been massively [failing](https://github.com/stevearc/pypicloud/actions) 😅

Reply:
They were generated by the person that added Azure support in #241. I, unfortunately, have never used Azure. Maybe we could ask @terrycain if he has any pointers?

I'm also going to be traveling for the next 10 days or so, and will have severely limited availability. Will try to check in when I can.

Reply:
safe travels!

Reply:
I'm not going to be able to help much for the next week or so too. I no longer use Azure, though I _might_ be able to get some access to have a look. As for how the VCR stuff works, I can't quite remember but I'll have a look if I can get access to some blob storage.

Reply:
found this one https://docs.microsoft.com/en-us/azure/storage/common/storage-use-azurite?tabs=docker-hub#tabpanel_1_docker-hub

will give it a spin next week